lib/Onyx.ts

```diff
@@ -134,6 +137,20 @@ function set<TKey extends OnyxKey>(key: TKey, value: OnyxSetInput<TKey>): Promis
         delete OnyxUtils.getMergeQueue()[key];
     }
 
+    const skippableCollectionMemberIDs = OnyxUtils.getSkippableCollectionMemberIDs();
+    if (skippableCollectionMemberIDs.size) {
+        try {
+            const [, collectionMemberID] = OnyxUtils.splitCollectionMemberKey(key);
+            if (skippableCollectionMemberIDs.has(collectionMemberID)) {
+                // The key is a skippable one, so we set the new value to null.
+                // eslint-disable-next-line no-param-reassign
+                value = null;
```

Comment:
Maybe we can get rid of param reassign error?
```
function set<TKey extends OnyxKey>(key: TKey, initialValue: OnyxSetInput<TKey>): Promise<void> {
   let value = initialValue;
```

```suggestion
                value = null;
```

Or is it you intention to reassign the value param?

Reply:
**UPD:** It looks like in similar cases in the codebase, the eslint-disable-next-line no-param-reassign is used as well. Considering this we can keep it as is.